### DISCUSSION TITLE
Submitted openrocket to Windows packaging repositories

### DISCUSSION THREAD

**Discussion by luzpaz** [hydrogen-music/hydrogen#1817] [Hydrogen on Windows]
- [ ] Chocolatey https://github.com/chocolatey-community/chocolatey-package-requests/issues/1434
- [ ] Winget https://github.com/microsoft/winget-pkgs/issues/112758
- [ ] ~Mingw https://github.com/msys2/MINGW-packages (any volunteers?)~

**Comment by theGreatWhiteShark** [comment]
Hey @luzpaz,

Thanks a lot!

> Winget https://github.com/microsoft/winget-pkgs/issues/112758

I was not even aware of this one.

> Mingw https://github.com/msys2/MINGW-packages (any volunteers?)

Do you think obtaining Hydrogen from MinGW is a relevant use case? Apart from the Windows pipeline of Hydrogen itself I have no exposure to Windows at all. But MinGW felt more like a developer thing whereas Choco felt definitely like a user thingy. Also, their pipeline seem to work by compiling the application from source. Chocolatey and Winget appear to just wrap the binaries we create in our build pipeline.

**Reply by luzpaz** [reply]
> Do you think obtaining Hydrogen from MinGW is a relevant use case? 

Probably not. Lets drop MinGW to lower the complexity.
